wellcad/com/_borehole.py

```diff
@@ -1256,6 +1283,57 @@ def calculate_borehole_volume(self, prompt_user=None, config=None):
         config : str, optional
             Path to a configuration file or a parameter string. The
             configuration file can contain the following options:
+            Note: This configuration is for WellCAD version 5.7 and onwards. For prior version, see the next section.
+
+            .. code-block:: ini
+
+                [VolumeProcess]
+                'OuterDiam, InnerDiam : name of a well log, img log or a constant value
+                'OuterDiamUnit, InnerDiamUnit : inch or mm, default to mm
+                'OuterDiam1, InnerDiam1 : name of a well log, img log or a constant value. Use either OuterDiam, InnerDiam or OuterDiam1, InnerDiam1 but not both !
+                'OuterDiamUnit1, InnerDiamUnit1 : inch or mm, default to mm. Use either OuterDiamUnit, InnerDiamUnit or OuterDiamUnit1, InnerDiamUnit1 but not both !
+                'OuterDiam2, InnerDiam2 : name of a well log
+                'OuterDiamUnit2, InnerDiamUnit2 : inch or mm, default to mm
+                'IntervalRef : name of a well log or value (if value, in depth units of the borehole doc)
+                'DepthRange : Maximum, UserDefined, Zones, LogZones
+                'LogZonesDepthRange=logname, depthsectionName1, depthsectionName2, ....depthsectionname3
```

Comment:
The lines of comments above are too long and bleed out of the rendered documentation. Also, comment lines should begin with `; `.